### DISCUSSION TITLE
a question

### DISCUSSION THREAD

**Discussion by huangchanghuai** [fastapi/fastapi#6825] [Questions] [answered]
Why don't we let go of "golang" when Python's fastapi is constrained by data types and other things that are not aesthetic or concise

**Comment by Mause** [comment]
Huh? What is the context for this question? What does go have to do with FastAPI?

**Comment by tiangolo** [comment, marked as answer]
Thanks for the help here @Mause ! 👏  🙇 

Sorry, I can't help you if you don't follow the issue template @huangchanghuai. I'm also not sure what you really mean or what you are asking, but you are of course welcome to go and work on Go lang.

If you have any other questions please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.
